src/sonic-config-engine/portconfig.py

```diff
@@ -88,11 +106,18 @@ def get_port_config(hwsku=None, platform=None, port_config_file=None, hwsku_conf
             port_alias_map = {}
             port_alias_asic_map = {}
             for intf_name in ports.keys():
-                port_alias_map[ports[intf_name]["alias"]] = intf_name
+                if "alias" in ports[intf_name]:
+                    port_alias_map[ports[intf_name]["alias"]] = intf_name
             return (ports, port_alias_map, port_alias_asic_map)
 
+    if asic_name is not None:
+        asic_id = str(get_asic_id_from_name(asic_name))
+    else:
+        asic_id = None
+
     if not port_config_file:
-        port_config_file = device_info.get_path_to_port_config_file(hwsku, asic)
+        port_config_file = device_info.get_path_to_port_config_file(hwsku, asic_id)
```

Comment:
in 201911 and here, we pass "asic_id" like 0/1 to get_path_to_port_config_file() ; That part is not modified.
The change here is that, portconfig.py consumes asic_name instead of asic_id and we get asic_id from asic_name here instead of getting that information in minigraph.py